workers/unity/Packages/io.improbable.gdk.tools/CodeGenDocs/faq.md

```diff
@@ -0,0 +1,81 @@
+# Frequently Asked Questions
```

Comment:
How many times have you been asked each of these questions 😛

Reply:
I feel like these are the questions I ended up asking as I was going through the rest of the pages, some questions that were left unanswered